watcher/watcher.go

```diff
@@ -48,7 +45,7 @@ func (w *FSWatcher) Watch(path string) error {
 	// this can never return false since we will always
 	// have at least one file in the directory (.snag.yml)
 	_ = w.watch(path)
-	w.ex.Send(rebuild, nil)
+	w.Event <- struct{}{}
```

Comment:
Should we do a non blocking wrote here? This could potentially block the watcher indefinitely.